.jenkinsci/bindings.groovy

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env groovy
+
+def doBindings() {
+
+    def cmake_options = ""
+    if (params.JavaBindings) {
+        cmake_options += " -DSWIG_JAVA=ON "
+    }
+    if (params.PythonBindings) {
+        cmake_options += " -DSWIG_PYTHON=ON "
+    }
+    // In case language specific options were not set,
+    // build for every language
+    if (!params.JavaBindings && !params.PythonBindings) {
+        cmake_options += " -DSWIG_JAVA=ON -DSWIG_PYTHON=ON "
+    }
+    sh """
+        cmake \
+          -H. \
```

Comment:
Fixed, but anyhow, as I mentioned in PR message, bindings cannot be built at the moment since on some platforms (particularly on Mac) JAVA_HOME env variable is always undefined for some reason